apps/src/code-studio/hiddenStageRedux.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Reducer and actions for stage lock info. This includes the teacher panel on
+ * the course overview page, and the stage locking dialog.
+ */
+
+import { INIT_PROGRESS } from './progressRedux';
+import experiments from '@cdo/apps/experiments';
+
+export const TOGGLE_HIDDEN = 'hiddenStage/TOGGLE_HIDDEN';
+
+const hiddenStagesEnabled = experiments.isEnabled('hiddenStages') || IN_UNIT_TEST;
```

Comment:
Could turn this into a function/getter and then your unit tests could just enable/disable the experiment themselves - unless you think there's a performance implication?